minetest.conf.example

```diff
@@ -1947,6 +1947,11 @@
 #    type: bool
 # high_precision_fpu = true
 
+#    -   Full (default):  Multiple singleplayer worlds, subgame choice, texture pack chooser, etc.
+#    -   Simple: One singleplayer world, no subgame or texture pack choosers. May be necessary for smaller screens.
+#    type: enum
+main_menu_style = full
```

Comment:
According to `settingtypes.txt` the default is `default`, where you use the default value `full` here. Please check what's actually the default value and correct the docs to match both the same value.